force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -83,7 +94,12 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the number is even, False otherwise.
      */
     public static Boolean isEven(Integer num) {
-        return null; // Replace null with the variable you used to store the result
+        //check if number is even or odd using math.mod() is equal to zero 
+        //if equals to zero, return Boolean value of true
+        if (Math.mod(num, 2) == 0){
+            return True; 
+        }
+        return False; // Replace null with the variable you used to store the result
```

Comment:
```suggestion
            return true; 
        }
        return false; // Replace null with the variable you used to store the result
```

true and false reserved words are lowercase

Reply:
mental note to remember that reserved words should be in lowercase (update completed)